### PR TITLE
CIV-1556 Refunds UI Tests

### PR DESCRIPTION
**Pull request by krishnanuthalapati**
Verified the tests in preview pipeline but as they are taking 3 mins of time , we dont want to increase the time in main pipeline so moving them to nightly and these tests can be used for regression
https://tools.hmcts.net/jira/browse/CIV-1556

### CODE REVIEW

File: e2e/api/steps.js [hunk 1]
```diff
@@ -166,6 +171,77 @@ let caseData = {};
...
+        microservice: config.s2s.microservice,
+        oneTimePassword: totp(config.s2s.secret)
+      })
+      .then(response => response.text());
+    const accountNumber = 'PBA0088192';
+
+    const saveBody = {
+      account_number: `${accountNumber}`,
+      amount: amount,
+      case_reference: '1253656',
```
Comment (mounikahmcts) [inline e2e/api/steps.js:192]: why are we passing dummy number?

File: e2e/api/steps.js [hunk 2]
```diff
@@ -166,6 +171,77 @@ let caseData = {};
...
+          calculated_amount: amount,
+          code: `${feeCode}`,
+          fee_amount: amount,
+          version: version,
+          volume: volume
+        }
+      ],
+      organisation_name: 'string',
+      service: 'CIVIL',
+      site_id: 'ABA6'
```
Comment (mounikahmcts) [inline e2e/api/steps.js:208]: civil site id is AAA7

File: e2e/tests/ui_tests/refunds/ui_request_for_refunds_test.js [hunk 3]
```diff
@@ -0,0 +1,53 @@
+const config = require('../../../config.js');
+const {unAssignAllUsers} = require('../../../api/caseRoleAssignmentHelper');
+const RequestRefundSteps = require('./steps/requestRefundSteps');
+
+Feature('Request and Processing of refunds').tag('@refunds');
+
+Scenario('Request and Approval of a Refund after a return to Requestor by the Approval Caseworker',
+  async ({I, api}) => {
+  if (['preview', 'demo', 'aat'].includes(config.runningEnv)) {
```
Comment (mounikahmcts) [inline e2e/tests/ui_tests/refunds/ui_request_for_refunds_test.js:9]: remove this if condiiton as it is applicable in all envs

File: e2e/api/steps.js [hunk 4]
```diff
@@ -166,6 +171,77 @@ let caseData = {};
 let mpScenario = 'ONE_V_ONE';
 
 module.exports = {
+
+  createAPBAPayment: async function(user, ccdCaseNumber, amount, feeCode, version, volume) {
```
Comment (mounikahmcts) [inline e2e/api/steps.js:175]: Move this method to apirequest.js

File: e2e/pages/refunds/requestRefund/refundCheckYourAnswers.page.js [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+    if (!changeReason) {
+      I.see('Payment amount');
+      I.see('£550');
+      I.see('Reason for refund');
+      if (!changeReason) {
+        I.see('Amended claim');
+      } else {
+        I.see('System/technical error');
+      }
+      }
```
Comment (mounikahmcts) [inline e2e/pages/refunds/requestRefund/refundCheckYourAnswers.page.js:16]: is this } required

**Review by mounikahmcts — APPROVED** [review]

**Review by rajakm — APPROVED** [review]

File: e2e/pages/refunds/requestRefund/refundCheckYourAnswers.page.js (hunk 5 above)
Comment (pats-john) [reply e2e/pages/refunds/requestRefund/refundCheckYourAnswers.page.js]: Removed.

**Pull request merged**